### ISSUE TITLE
Provide helpful message at the first run when configurations are still empty

### ISSUE THREAD

**Issue by Pinolo** [sparkfabrik/sparktool#7]
example output when configuration file is empty:

```
% ./spark.php redmine:search



  [Exception]
  <url> malformed



redmine:search [--report] [-l|--limit [LIMIT]] [-s|--sort [SORT]] [-p|--project_id [PROJECT_ID]] [--status [STATUS]] [--assigned [ASSIGNED]] [--sprint [SPRINT]] [--created [CREATED]] [--updated [UPDATED]] [--not-estimated] [--subject [SUBJECT]] [--tracker [TRACKER]]
```

**Comment by grota** [comment]
Duplicate of #11

**Comment by grota** [comment]
Fixed with #32
